load-gen/src/main/java/io/smartcat/data/loader/api/RateGenerator.java

```diff
@@ -0,0 +1,6 @@
+package io.smartcat.data.loader.api;
+
+public interface RateGenerator {
+
+    long getRate(long prevInstant, long currentInstant);
```

Comment:
Actually, do we need linar or just some periodical. Let say sine
getRate(x) { return sin(x);}